lp-collaborative-workspace/lp-cw-bridge/lp-cw-bridge-transformer/resources/transformation/ado2xwiki.atl

```diff
@@ -271,6 +291,7 @@ rule MODELType2WebHome {
 		type <- 'StaticList',
 		value <- s.modeltype.mapStandardModelType
 	)
+	--aggiungere una property documentation
```

Comment:
Same here, there is a property `documentation` that you can use in the object `LPCode.ModelClass`, I've checked.  By the way, please try to write comments in English ;-)